src/element-events.js

```diff
@@ -51,8 +53,11 @@ export class ElementEvents {
    * Adds and Event Listener on the context element.
    * @return Returns the eventHandler containing a dispose method
    */
-  subscribe(eventName: string, handler: Function, captureOrOptions?: boolean = true): EventHandler {
+  subscribe(eventName: string, handler: Function, captureOrOptions?: boolean | AddEventListenerOptions): EventHandler {
     if (typeof handler === 'function') {
+      if (typeof captureOrOptions === 'undefined') {
```

Comment:
I'll repeat what @jdanyow said to me about this case: the rest of the code base doesn't care about crazy people changing default value of `undefined`, so lets just compare `captureOrOptions === undefined`

Reply:
ok changed that